packages/voorbeeld-design-tokens/documentation/number-badge/utrecht.numberBadge.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import { Meta, StoryObj } from '@storybook/react';
+import { BadgeCounter } from '@utrecht/component-library-react/dist/css-module';
+
+const meta = {
+  id: 'utrecht-badge-counter-group',
+  title: 'Components/Number Badge/Utrecht',
+  component: BadgeCounter,
+  render: () => (
```

Comment:
In dit geval heb je geen `render` function nodig maar kun je de `9` gewoon mee geven als `children` onder `args`